src/components/meetupSection.js

```diff
@@ -36,6 +34,11 @@ const MeetupSection = () => {
     },
   } = useStaticQuery(facebookEventQuery)
 
+  const upcomingEvents = data.filter(event =>
+    isFuture(parseISO(event.start_time))
+  )
+  const pastEvents = data.filter(event => isPast(parseISO(event.start_time)))
```

Comment:
How will this handle events that are on the current day?

Reply:
Time can tell, an event that falls on the current day but not yet started is still an upcoming event but if it's already started, it will consider as a past event.

Also, note that this will not change/update automatically, we still need to rebuild the site if we add future events or if the event is over. We need to figure out how we can automate this.

Reply:
> We need to figure out how we can automate this.

I'll merge this, but please look into it. Thanks! :bow: